src/System.Management.Automation/resources/DebuggerStrings.resx

```diff
@@ -249,4 +249,13 @@ The current session does not support debugging; operation will continue.
   <data name="InvalidDetachCommand" xml:space="preserve">
     <value>The debugger detach command is not applicable.  The detach command only applies when debugging jobs and runspaces with the Debug-Job or Debug-Runspace cmdlets.</value>
   </data>
+  <data name="InvalidRunspaceId" xml:space="preserve">
+    <value>Invalid runspace id.</value>
```

Comment:
Do you think there is value here in also emitting the runspaceId which was invalid?

Reply:
added.